cell.go

```diff
@@ -58,7 +58,7 @@ func (cb *CellBuffer) SetContent(x int, y int,
 		// dirty as well as the base cell, to make sure we consider
 		// both cells as dirty together.  We only need to do this
 		// if we're changing content
-		if (c.width > 0) && (mainc != c.currMain || !reflect.DeepEqual(combc, c.currComb)) {
+		if (c.width > 0) && (mainc != c.currMain || !(len(combc) > 0 && len(combc) == len(c.currComb) && reflect.DeepEqual(combc, c.currComb))) {
```

Comment:
I don't think this is quite right, although well intentioned.

Probably we want to make it something like:

```suggestion
		if (c.width > 0) && (mainc != c.currMain || len(combc) != len(c.currComb) || (len(combc) > 0 && !reflect.DeepEqual(combc, c.currComb))) {
```